Controllers/TimeWorkedAndVacationController.cs

```diff
@@ -45,48 +44,46 @@ public async Task<IActionResult> Details(int? id)
         }
 
         // GET: TimeWorkedAndVacation/Create
-        public IActionResult Create1()
+        public IActionResult Create()
         {
             return View();
         }
 
-        // POST: TimeWorkedAndVacation/Create
-        // To protect from overposting attacks, enable the specific properties you want to bind to, for 
-        // more details, see http://go.microsoft.com/fwlink/?LinkId=317598.
+
+
         [HttpPost]
         [ValidateAntiForgeryToken]
-        
-        /*public async Task<IActionResult> Create()
+        public  ActionResult Create(TotalNumberOfDaysWorked totalNumberOfDaysWorked, TotalNumberOfAvailableVacationDays totalNumberOfAvailableVacationDays)
         {
-            var TotalNumberOfDaysWorked = 0;
-            var TimeWorkedAndVacation = 0.0125 * (double) TotalNumberOfDaysWorked;
-            
+            totalNumberOfAvailableVacationDays = totalNumberOfDaysWorked * .0125;
```

Comment:
Clarification? This method ideally should not be in this controller but in a separate controller? So that it is more accessible to the entire project?